src/pyatsimagebuilder/builder.py

```diff
@@ -434,5 +436,28 @@ def _build_image(self, no_cache=False):
         if not self.image.id:
             # we've failed to set the image id - something is wrong!
             raise Exception('No confirmation of successful build.')
-    
 
+    def _replace_environment_variables(self):
+
+        ENV_PATTERN = re.compile(r'(%ENV{ *([0-9a-zA-Z\_]+) *})')
```

Comment:
make this a global at the top of the file

Reply:
done